src/main/resources/jenkins/plugins/ui_samples/TextBox/index.jelly

```diff
@@ -0,0 +1,74 @@
+<!--
+  ~ The MIT License
+  ~
+  ~ Copyright (c) 2010, InfraDNA, Inc.
+  ~
+  ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+  ~ of this software and associated documentation files (the "Software"), to deal
+  ~ in the Software without restriction, including without limitation the rights
+  ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  ~ copies of the Software, and to permit persons to whom the Software is
+  ~ furnished to do so, subject to the following conditions:
+  ~
+  ~ The above copyright notice and this permission notice shall be included in
+  ~ all copies or substantial portions of the Software.
+  ~
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  ~ THE SOFTWARE.
+  -->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:s="/lib/samples">
+  <s:sample>
+    <div class="app-component-sample">
+      <f:entry title="Example" field="example">
+        <f:textbox />
```

Comment:
There's a styling issue here where the textbox isn't taking up the appropriate width for the form

<img width="452" alt="image" src="https://user-images.githubusercontent.com/21194782/159113300-fb84324d-235d-4dd0-a6ba-19ffe4912739.png">

I haven't managed to track down why the width isn't being applied.
As far as I can tell the main-panel sets a width of 100% and then the form-item sets a max width of smaller but still a lot bigger than this.

If I set the width explicitly on the form-item to 100% it works.

I'm wondering if `app-component-sample` is affecting it somehow